packages/react-hooks/src/ConnectionProvider.tsx

```diff
@@ -27,7 +32,7 @@ export const useConnectionById = (id: string): ConnectionRecord | undefined => {
 export const useConnectionByState = (state: ConnectionState): ConnectionRecord[] => {
```

Comment:
Maybe we can update this to `DidExchangeState`? That's what we're using now for all connections regardless of protocol so we can work with a singular set of states (instead of two sets with slightly different values). Then we won't need `rfc0160StateFromDidExchangeState`